api/Makefile

```diff
@@ -78,8 +78,16 @@ gen-files .generate_files: lint-cache-dir clean-generated
 	   sh -c '$(GIT_CONFIG_SSH) openapi-gen \
 		--v 1 --logtostderr \
 		--go-header-file "/go/src/$(PACKAGE_NAME)/hack/boilerplate/boilerplate.go.txt" \
-		--input-dirs "$(PACKAGE_NAME)/pkg/apis/projectcalico/v3,k8s.io/api/core/v1,k8s.io/api/networking/v1,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/version,k8s.io/apimachinery/pkg/runtime,k8s.io/apimachinery/pkg/util/intstr,$(PACKAGE_NAME)/pkg/lib/numorstring" \
-		--output-package "$(PACKAGE_NAME)/pkg/openapi"'
+		--output-dir "/go/src/$(PACKAGE_NAME)/pkg/openapi" \
```

Comment:
`--output-dir` flag is enforced in https://github.com/kubernetes/code-generator/commit/0723f185982784c093f7925c4749d984e9fc0ad9.